infra/identity/prod/westeurope/outputs.tf

```
@@ -0,0 +1,13 @@
+output "managed_identity_github_ci" {
+  value = {
+    name                = module.managed_identities.github_identity_ci.name
+    resource_group_name = module.managed_identities.github_identity_ci.resource_group_name
+  }
+}
+
+output "managed_identity_github_cd_name" {
```

```suggestion
output "managed_identity_github_cd" {
```

why do we have 'name' suffix above but not here?

---

thanks for catching that